database/staking.go

```diff
@@ -97,35 +156,10 @@ func (db BigDipperDb) SaveValidatorsData(validators []types.Validator) error {
 	return err
 }
 
-// GetValidatorsData returns the data of all the validators that are currently stored inside the database.
-func (db BigDipperDb) GetValidatorsData() ([]types.Validator, error) {
-	sqlStmt := `SELECT DISTINCT ON (validator.consensus_address)
-				validator.consensus_address, validator.consensus_pubkey, validator_info.operator_address 
-				FROM validator 
-				INNER JOIN validator_info 
-				ON validator.consensus_address = validator_info.consensus_address
-				ORDER BY validator.consensus_address`
-
-	var rows []dbtypes.ValidatorData
-	err := db.Sqlx.Select(&rows, sqlStmt)
-	if err != nil {
-		return nil, err
-	}
-
-	validators := make([]types.Validator, len(rows))
-	for index, row := range rows {
-		validators[index] = row
-	}
-
-	return validators, nil
-}
-
-// _________________________________________________________
-
```

Comment:
Why was this removed entirely? Can we bring it back?

Reply:
Duplicate with GetValidatorsData()